<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWin</class>
 <widget class="QMainWindow" name="ClientWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 0);</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>1920</width>
      <height>1010</height>
     </rect>
    </property>
    <widget class="QWidget" name="daily_journal_page_0">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1010</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_DJView">
       <attribute name="title">
        <string>View</string>
       </attribute>
       <widget class="QDateEdit" name="dateEdit_DJView">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>0</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_DJView">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>40</y>
          <width>960</width>
          <height>620</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_DJEdit">
       <attribute name="title">
        <string>Edit</string>
       </attribute>
       <widget class="QDateEdit" name="dateEdit_DJEdit">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>0</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_DJEdit">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>40</y>
          <width>960</width>
          <height>620</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_DJESave">
        <property name="geometry">
         <rect>
          <x>1550</x>
          <y>760</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Save Changes</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="eb_page_1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1920</width>
        <height>1010</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>View</string>
       </attribute>
       <widget class="QDateEdit" name="dateEdit_EBView">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>0</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Edit</string>
       </attribute>
       <widget class="QDateEdit" name="dateEdit_EBEdit">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>0</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="ChangePassword">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>1480</x>
        <y>0</y>
        <width>321</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Old Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>New Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Confirm Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Save Changes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Daily Journal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Expenditure Book</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ChangePassword">
    <property name="geometry">
     <rect>
      <x>1642</x>
      <y>40</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Logout">
    <property name="geometry">
     <rect>
      <x>1820</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
